src/ray/raylet/scheduling/cluster_task_manager_test.cc

```diff
@@ -132,20 +140,25 @@ class ClusterTaskManagerTest : public ::testing::Test {
     node_info_[id] = info;
   }
 
+  void AssertNoLeaks() {
+    ASSERT_TRUE(task_manager_.tasks_to_schedule_.empty());
+    ASSERT_TRUE(task_manager_.tasks_to_dispatch_.empty());
+    ASSERT_TRUE(task_manager_.waiting_tasks_.empty());
+    ASSERT_TRUE(task_manager_.infeasible_tasks_.empty());
```

Comment:
Should we also check the task dep manager has everything unsubscribed?

Reply:
Good idea, I'll add that to the mock.